components/ToggleButton.js

```diff
@@ -23,27 +23,24 @@ const breakpoint = 768,
   teal = '#23485f'
 
 const Button = styled.button`
+  @apply px-4 py-2 border-0 text-white;
   appearance: none;
-  color: white;
-  padding: 15px;
-  border: none;
 
   background: ${props => props.isActive ? teal : 'black' };
   &:hover {
     background: ${props => lighten(.1, props.isActive ? teal : 'black') };
   }
 
   @media (min-width: ${breakpoint + 1}px) {
-    font-size: 1.25em;
+    @apply text-lg;
   }
   @media (max-width: ${breakpoint}px) {
-    padding: 10px;
+    @apply px-2 py-1;
   }
 
   /* Not possible in vanilla-extact (though not really a good practice ) */
   > span {
-    display: inline-block;
-    transition: transform .2s ease-out;
+    @apply inline-block transition-all;
```

Comment:
This is a minor thing(and you probably know it), but If the transition is only for transform then:
```suggestion
    @apply inline-block transition-transform;
```

Reply:
I didn't know about that and I didn't love that I'd made it so broad 👍